src/v2/Components/v2/ArtworkFilter/ArtworkFilters/SizeFilter2.tsx

```diff
@@ -40,6 +40,28 @@ const parseRange = (range?: string) => {
   })
 }
 
+const convertRangeElementToCentimeters = (bar: number | "*") => {
```

Comment:
`bar` seems like an odd variable name, maybe `val` or something?

Reply:
Thanks for catching this. It was a left over from `const foo = (bar) => {}`